src/server/gateways/postgres/plugin.ts

```diff
@@ -22,20 +21,16 @@ export default class PostgresGateway extends Gateway {
     });
   }
 
-  public async read(id: string): Promise<any> {
+  public async readById(id: string): Promise<any> {
     const client: PoolClient = await this.pool.connect();
 
     try {
       const queryText: string = `SELECT * FROM trades WHERE id = '${id}'`;
 
-      const response: QueryResult = await client.query(queryText, [id]);
-
-      await client.query(TRANSACTIONS.COMMIT);
```

Comment:
**Annotation:** We aren't doing complex graph queries at the moment, so we can do away with the commits and rollbacks.